swig/helics.i

```diff
@@ -17,10 +17,13 @@
 %apply helics_time_t *OUTPUT {helics_time_t*};
 %apply (char *STRING, size_t LENGTH) { (const char *data, int len) };
 %cstring_output_maxsize(char *str, int maxlen);
+%cstring_bounded_output(char* str, 1024);
```

Comment:
cstring stuff has to be in the helicsPython.i since java doesn't work with cstring.i